src/main/java/com/diffplug/gradle/pde/PdeBuildTask.java

```diff
@@ -88,6 +90,17 @@
  * ```
  */
 public class PdeBuildTask extends DefaultTask {
+	private WorkspaceRegistry registry;
+
+	public PdeBuildTask() {
+		try {
+			this.registry = WorkspaceRegistry.instance();
+		} catch (IOException e) {
+			getLogger().warn("WorkspaceRegistry could not be instantiated! Using default workspace.", e);
+			this.registry = null;
+		}
+	}
+
 	private Object destination;
```

Comment:
I try to avoid state and config whenever possible.  I would remove all this config, and just assume that `WorkspaceRegistry.instance()` will always succeed.